packages/webapp/src/views/developer/details-email-confirm.html

```diff
@@ -0,0 +1,86 @@
+
+{% extends 'developer-form-layout.html' %}
+
+{% set pageHeading = "Is this email address correct?" %}
+
+{% block formContent %}
+<div class="govuk-body">
+    {% if err %}
+        {% include "errorbanner.html" %}
+    {% endif %}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            {% set emailHtml %}
+                {% if errorMessage %}
```

Comment:
Don't think it needs the if statement here just to set the errorMessage, errorMessage could just be set as an empty object/array if not in error